packages/shared/lib/services/connection.service.ts

```diff
@@ -58,6 +59,7 @@ import {
     extractValueByPath,
     stripCredential
 } from '../utils/utils.js';
+import { flushLogsBuffer } from '@nangohq/logs';
```

Comment:
I don't think `logs` is a dependency of `shared` 😢

Reply:
aaah true, I was wondering why the CLI tests was failing, thanks